authentification/transport/http/login.go

```diff
@@ -0,0 +1,35 @@
+package http
+
+import (
+	"context"
+	"net/http"
+	"time"
+
+	"log"
+
+	"github.com/There-is-Go-alternative/GoMicroServices/authentification/usecase"
+	"github.com/gofiber/fiber"
+)
+
+func LoginHandler(cmd usecase.LoginProto) fiber.Handler {
+	return func(c *fiber.Ctx) {
+		var dto usecase.LoginDTO
+
+		ctx, cancel := context.WithTimeout(context.Background(), 100*time.Second)
```

Comment:
Tu peux récupérer un context fastHttp depuis le context fiber qui est compatible avec l'interface context.Context:
https://docs.gofiber.io/api/ctx#context

```suggestion
		ctx, cancel := context.WithTimeout(c.Context() 100*time.Second)
```